etc/config/c++.amazon.properties

```diff
@@ -3506,6 +3506,15 @@ libs.gemmlowp.versions=trunk
 libs.gemmlowp.versions.trunk.version=trunk
 libs.gemmlowp.versions.trunk.path=/opt/compiler-explorer/libs/gemmlowp/trunk
 
+libs.glaze.name=glaze
+libs.glaze.description=JSON, reflection, and interface library
+libs.glaze.versions=trunk:198
+libs.glaze.url=https://github.com/stephenberry/glaze
+libs.glaze.versions.trunk.version=trunk
+libs.glaze.versions.trunk.path=/opt/compiler-explorer/libs/glaze/trunk
```

Comment:
There's no trunk glaze installed in your infra PR; there's just the `v1.9.8`. I'll quickly hack that in too :)

Currently...
```
$ ls /opt/compiler-explorer/libs/glaze/
v1.9.8
```

Reply:
now installed!